release-content/0.14/release-notes/10690_Upgrade_to_Taffy_04.md

```diff
@@ -1 +1,5 @@
-TODO
+[`taffy`](https://crates.io/crates/taffy), the UI layout crate that we rely on, has had a major update.
+In addition to various bug fixes, this brings two new pieces of functionality to users of `bevy_ui`:
+
+1. `Display::Block`: use the [CSS block layout model](https://www.w3schools.com/css/css_display_visibility.asp) to layout UI nodes. Elements will start on a new line, and take up the whole width. Great for in-line images!
```

Comment:
This is not great for inline images! We need "inline" layout for that. This just does vertical block layout. IMO the primary use case would be matching the layout of an existing UI ported from web. Otherwise you're better off using Flexbox.

Reply:
For inline we'll need something like https://github.com/linebender/parley/pull/67 (or similar support in cosmic text)